Add AvaTax troubleshooting section (#1066)
saleor/saleor-docs: docs/developer/app-store/apps/taxes/avatax/overview.mdx

:::info
The value of the `avataxTaxCalculationDate` field must be a valid UTC datetime string.
:::

## Troubleshooting

### "I configured the app, but the tax value is 0"

Creating an AvaTax provider and assigning it to a channel ends the app configuration process. The taxes should be calculated automatically from now on.

> If that doesn't happen, it's possible that it's an expected AvaTax behavior. There is a number of reasons why AvaTax can return 0 as the tax value and they are all described in the "[Reasons Tax Can Be Zero](https://www.avalara.com/partner/en/api/calculation/sales-tax/guide/exemptions/reasons-tax-can-be-zero.html)" article.

Reviewer: ```suggestion
If that doesn't happen, it may be an expected AvaTax behavior. There are several reasons why AvaTax can return 0 as the tax value and they are all described in the "[Reasons Tax Can Be Zero](https://www.avalara.com/partner/en/api/calculation/sales-tax/guide/exemptions/reasons-tax-can-be-zero.html)" article.
```
